integration/dubbo-alibaba/src/main/java/io/seata/integration/dubbo/alibaba/TransactionPropagationFilter.java

```diff
@@ -41,19 +40,24 @@
     @Override
     public Result invoke(Invoker<?> invoker, Invocation invocation) throws RpcException {
         String xid = RootContext.getXID();
+        String xidInterceptorType = RootContext.getXIDInterceptorType();
+
         String rpcXid = RpcContext.getContext().getAttachment(RootContext.KEY_XID);
+        String rpcXidInterceptorType = RpcContext.getContext().getAttachment(RootContext.KEY_XID_INTERCEPTOR_TYPE);
```

Comment:
the `RootContext.KEY_XID_INTERCEPTOR_TYPE` may be appear lowcase. issue #1728 . The suggestion is to use lowercase at the beginning. `tx-xid-interceptor-type`